src/components/radio-group/radio-group-component.jsx

```diff
@@ -1,39 +1,23 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-import InfoModal from 'components/modal';
-import { layersConfig } from 'constants/mol-layers-configs';
+import { ReactComponent as InfoIcon } from 'icons/info.svg';
 
 import styles from './radio-group-styles.module.scss';
 
 const RARITY = 'rarity';
 const RICHNESS = 'richness';
 
-const RadioGroup = ({ activeLayers, options, title, handleSimpleLayerToggle, handleExclusiveLayerToggle }) => {
-  const selectedLayersTitles = activeLayers
-        .map(l => layersConfig.find(lc => lc.slug === l.id))
-        .filter(l => l)
-        .map(l => l.title);
+const RadioGroup = ({ activeLayers, options, title, handleSimpleLayerToggle, handleExclusiveLayerToggle, handleInfoClick }) => {
 
-  const optionsLayers = [];
-  options.forEach(option => {
-    Object.keys(option.layers).forEach(variant => {
-      optionsLayers.push({
-        layerTitle: option.layers[variant],
-        variant,
-        option
-      });
-    });
-  });
+  const [variant, setVariant] = useState(RARITY);
```

Comment:
Hmm, that won't work. As the selected variant source of truth is the active layer that info should be taken from activeLayers. When this component is destroyed we will use that inner state. Try to close Biodiversity Layers and open them again. You will see that it is not selecting radio buttons correctly.